### PR TITLE
fettuccine alfredo recipe

### PR DESCRIPTION
**Pull request by diego-codes**

### CODE REVIEW

File: pastas/fetuccinne-alfredo.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+## Ingredients
+- 12 ounces fettuccine or other long pasta (Preferrably fresh pasta)
+- Kosher salt
+- 1/4 cup (1/2 stick) unsalted butter
+- 3/4 cup finely grated Parmesan, plus more for serving
+- Freshly ground black pepper
+
+## Instructions
+1. In a large pot, boil water with a good amount of salt.
```
Comment (NicMoe) [inline pastas/fetuccinne-alfredo.md]: How much salt? This isn't our Italian nonna's verbal recipe, use actual quantities.

File: pastas/fetuccinne-alfredo.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+2. When the water is boiling, add (12 ounces) pasta and cook according to the instructions on the container. Cook for the lowest amount of time required. The pasta will finish cooking in the sauce laster.
```
Comment (NicMoe) [inline pastas/fetuccinne-alfredo.md]: What is "sauce laster"?

File: pastas/fetuccinne-alfredo.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+3. Reserve 2 cups of the pasta cooking water when the pasta is almost done.
+
+4. Drain the pasta and set aside while the sauce is made.
+
+5. In a large skillet on medium low heat, pour 1 cup of the reseved pasta cooking water. Get it so a simmer.
```
Comment (NicMoe) [inline pastas/fetuccinne-alfredo.md]: "Get it so a simmer"?

File: pastas/fetuccinne-alfredo.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+6. Actively whisk/mix the butter ½ to 1 tbsp at a time so the butter and the water emulsify (combine into homogenous liquid). Wait until the piece of butter is fully incorporated into the sauce before adding the next piece.
```
Comment (NicMoe) [inline pastas/fetuccinne-alfredo.md:23]: I appreciate that you defined "emulsify" here, I don't think I'd know what that means.

File: pastas/fetuccinne-alfredo.md (hunk 3 above)
Comment (NicMoe) [reply pastas/fetuccinne-alfredo.md:21]: Also, "reseved"?

Comment (NicMoe) [inline pastas/fetuccinne-alfredo.md]:
```suggestion
5. In a large skillet on medium low heat, pour 1 cup of the reserved pasta cooking water. Get it so a simmer.
```

File: pastas/fetuccinne-alfredo.md (hunk 2 above)
Comment (diego-codes) [reply pastas/fetuccinne-alfredo.md]: I meant later

File: pastas/fetuccinne-alfredo.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# Fettuccine alfredo
+
+Traditionally, this dish does not use cream, just lots of butter and parmesan. This recipe is a combination of a [Bon Appetit recipe](https://www.bonappetit.com/recipe/fettuccine-alfredo) and an [Epicurious video](https://www.youtube.com/watch?v=t81529ANEhI).
+
+## Ingredients
+- 12 ounces fettuccine or other long pasta (Preferrably fresh pasta)
```
Comment (NicMoe) [inline pastas/fetuccinne-alfredo.md]:
```suggestion
- 12 ounces fettuccine or other long pasta (preferably fresh pasta)
```

File: pastas/fetuccinne-alfredo.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+- Kosher salt
```
Comment (NicMoe) [inline pastas/fetuccinne-alfredo.md]: How much salt?

File: pastas/fetuccinne-alfredo.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+- 1/4 cup (1/2 stick) unsalted butter
+- 3/4 cup finely grated Parmesan, plus more for serving
```
Comment (NicMoe) [inline pastas/fetuccinne-alfredo.md:9]:
```suggestion
- 3/4 cup finely grated parmesan, plus more for serving
```

File: pastas/fetuccinne-alfredo.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+- Freshly ground black pepper
```
Comment (NicMoe) [inline pastas/fetuccinne-alfredo.md]: How much pepper?

File: pastas/fetuccinne-alfredo.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+## Instructions
+1. In a large pot, boil water with a good amount of salt (1 1/2 tbps).
```
Comment (NicMoe) [inline pastas/fetuccinne-alfredo.md]:
```suggestion
1. In a large pot, boil water with a good amount of salt (1 1/2 tablespoons).
```

File: pastas/fetuccinne-alfredo.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+
+3. Reserve 2 cups of the pasta cooking water when the pasta is almost done.
+
+4. Drain the pasta and set aside while the sauce is made.
+
+5. In a large skillet on medium low heat, pour 1 cup of the reserved pasta cooking water. Get it to a simmer.
+
+6. Actively whisk/mix the butter ½ to 1 tbsp at a time so the butter and the water emulsify (combine into homogenous liquid). Wait until the piece of butter is fully incorporated into the sauce before adding the next piece.
+
+7. Once all of the butter is incorporated into the sauce, add the parmesan cheese in parts, much like you did with the butter, one part at a time.
```
Comment (NicMoe) [inline pastas/fetuccinne-alfredo.md]:
```suggestion
7. Once all of the butter is incorporated into the sauce, add the Parmesan cheese in parts, much like you did with the butter, one part at a time.
```

File: pastas/fetuccinne-alfredo.md [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+
+8. After incorporating the ingredients for the sauce, add the cooked pasta and toss until the sauce blankets the pasta.
+
+9. Top with freshly ground black pepper and more grated parmesan. Enjoy!
```
Comment (NicMoe) [inline pastas/fetuccinne-alfredo.md]:
```suggestion
9. Top with freshly ground black pepper and more grated Parmesan. Enjoy!
```

**Comment by diego-codes** [comment]
@NicMoe this is ready for review again, thank you!

**Review by NicMoe — APPROVED** [review]

**Pull request merged**